src/components/modules/banners/contributions/ContributionsBanner.tsx

```diff
@@ -59,6 +68,89 @@ const styles = {
     ctaContainer: css`
         display: flex;
         justify-content: flex-end;
+        flex-direction: column;
+    `,
+    secondaryCtaContainer: css`
+        margin-left: ${space[4]}px;
+    `,
+    reminderContainer: css`
+        position: relative;
+        margin-top: ${space[2]}px;
+
+        ${from.tablet} {
+            margin-top: 0;
+        }
+    `,
+    reminderLine: css`
+        border-top: 1px solid black;
+        position: absolute;
+        top: 0;
+        right: 0;
+        left: 0;
+
+        ${from.leftCol} {
+            left: calc(50% - 550px + 1120px * 0.14285714285714285 - 20px + 10px + 1px);
+        }
+
+        ${from.wide} {
+            left: calc(50% - 630px + 1280px * 0.1875 - 20px + 10px);
+        }
+
+        &:before {
```

Comment:
The pseudo elements here are used to create the little caret. We have two: a bigger one that is black, and a smaller one that is the same yellow as the banner to give the effect of an outline

<img width="171" alt="Screenshot 2021-05-13 at 15 30 58" src="https://user-images.githubusercontent.com/17720442/118140571-62788f80-b400-11eb-95b6-25d8868333ec.png">